src/modules/user/openOrders/helpers.ts

```diff
@@ -83,7 +91,33 @@ export const insertOrUpdate = (list: OrderCommon[], order: OrderCommon): OrderCo
     }
 };
 
+export const insertOrUpdateFinex = (list: OrderCommon[], order: OrderCommon): OrderCommon[] => {
```

Comment:
please rename to insertOrUpdateByUUID
we might use UUID in peatio in the future